web/templates/status.html

```diff
@@ -56,7 +56,9 @@
                 </span>
               </td>
               <td>
-                {{stripLabels .BaseLabels "job" "instance"}}
+                <a href="#" data-toggle="tooltip" title="" data-original-title="Before Relabel: {{.MetaLabels}}">
```

Comment:
Hmm, with long DNS SRV names and such, the labels kind of overflow the tooltip and aren't completely visible. Over time, we should find a better visualization here.